ddtrace/internal/forksafe.py

```diff
@@ -55,3 +57,55 @@ def _after_fork():
         ddtrace_after_in_child()
 
     threading._after_fork = _after_fork  # type: ignore[attr-defined]
+
+
+_resetable_objects = weakref.WeakSet()  # type: weakref.WeakSet[ResetObject]
+
+
+def _reset_objects():
+    # type: (...) -> None
+    for obj in _resetable_objects:
+        try:
+            obj._reset_object()
+        except Exception:
+            log.exception("Exception ignored in object reset forksafe hook %r", obj)
```

Comment:
```suggestion
            log.exception("Exception ignored in object reset forksafe hook %r", obj, exc_info=True)
```
?

Reply:
`log.exception` 😉

Reply:
Ah sorry! In my mind that read `error` 😊